<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRun_DriversWindow</class>
 <widget class="QDialog" name="NewRun_DriversWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новий рейс : водії</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newrun_add_driver_pushButton">
       <property name="text">
        <string>Додати водія</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newrun_remove_driver_pushButton">
       <property name="text">
        <string>Відкріпити водія</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="newrun_submitdrivers_buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="newrun_driverswindow_title_lb">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Виберіть серед доступних водіїв</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="newrun_selected_drivers_lb">
     <property name="text">
      <string>Ви вибрали наступних водіїв:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="newrun_selected_drivers_comboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="newrun_Drivers_general_tableWidget">
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>№</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Прізвище</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ім'я</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Досвід</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Останній медогляд</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Заробітна плата</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableWidget" name="newrun_Drivers_runs_registered_tableWidget">
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>№</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Відправлення</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Прибуття</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Час відправлення</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Час прибуття</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="newrun_drivers_runs_registered_hide_pushButton">
     <property name="text">
      <string>Сховати</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
